apps/ledger-live-mobile/.env.android.staging

```diff
@@ -0,0 +1,5 @@
+APP_NAME="LL [STAGING]"
+SENTRY_DSN=https://0109819a39084e718120d031def0db38@o118392.ingest.sentry.io/6619343
+ANALYTICS_TOKEN=jfUZbw28ig8JpEi9DZpTUc21dKUKu1e3
```

Comment:
Do we need to put this as a secret in the CI ?

Reply:
no, these are not secrets. the .env at the llm level actually ends up in the mobile build.